src/metric/definition.go

```diff
@@ -206,6 +215,7 @@ var ControllerManagerSpecs = definition.SpecGroups{
 				ValueFunc: prometheus.FromValueWithOverriddenName(
 					"node_collector_evictions_total",
 					"nodeCollectorEvictionsDelta",
+					prometheus.IgnoreLabelsFilter("zone"),
```

Comment:
why this?

Reply:
I wanted to simply skip this label as it can have high cardinality...

Reply:
is this a new metric? if it is , it might have sense to ignore all the labels so the metric doesn't get renamed under any condition.

Reply:
For now I just leave it explicitly excluding the single label it was.. It's not clear (I'd say not possible) how to exclude all the labels for a metric.

Reply:
no sorry, 🤦 this was what we had before , jes.... this methods . Perhaps a comment there for the future us would be good in this case